src/plugins/orangehrmLDAPAuthenticationPlugin/Api/LDAPConfigAPI.php

```diff
@@ -129,96 +118,98 @@ public function create(): EndpointResult
             )
         );
 
-        $this->setConfigAttributes($LDAPSettings);
-        $this->getConfigService()->setLDAPSetting($LDAPSettings->getEncodedAttributes());
-        return new EndpointResourceResult(LDAPConfigModel::class, $LDAPSettings);
+        $this->setConfigAttributes($ldapSettings);
+        $this->getConfigService()->setLDAPSetting($ldapSettings);
+        return new EndpointResourceResult(LDAPConfigModel::class, $ldapSettings);
     }
 
     /**
-     * @param LDAPSetting $LDAPSetting
+     * @param LDAPSetting $ldapSetting
      */
-    private function setConfigAttributes(LDAPSetting $LDAPSetting): void
+    private function setConfigAttributes(LDAPSetting $ldapSetting): void
     {
         $bindAnonymously = $this->getRequestParams()->getBoolean(
             RequestParams::PARAM_TYPE_BODY,
             self::PARAMETER_BIND_ANONYMOUSLY
         );
-        $LDAPSetting->setBindAnonymously($bindAnonymously);
+        $ldapSetting->setBindAnonymously($bindAnonymously);
         if (!$bindAnonymously) {
-            $LDAPSetting->setBindUserDN(
+            $ldapSetting->setBindUserDN(
                 $this->getRequestParams()->getString(
                     RequestParams::PARAM_TYPE_BODY,
                     self::PARAMETER_DISTINGUISHED_NAME
                 )
             );
-            $LDAPSetting->setBindUserPassword(
-                $this->getRequestParams()->getString(
-                    RequestParams::PARAM_TYPE_BODY,
-                    self::PARAMETER_DISTINGUISHED_PASSWORD
-                )
+
+            $password = $this->getRequestParams()->getStringOrNull(
+                RequestParams::PARAM_TYPE_BODY,
+                self::PARAMETER_DISTINGUISHED_PASSWORD
             );
+            if (!is_null($password)) {
```

Comment:
@SandeepaOHRM If password is set already and user update other fields the password is not sent from FE. here is prevent updating password is null is sent from FE

Reply:
```php
if (!$bindAnonymously) {
    // ....
} else {
    $ldapSetting->setBindUserDN(null);
    $ldapSetting->setBindUserPassword(null);
}
```
:thinking: